wandb/cli/cli.py

```diff
@@ -423,7 +435,14 @@ def init(ctx, project, entity, reset, mode):
 
 
 @cli.command(
-    context_settings=CONTEXT, help="Upload an offline training directory to W&B"
+    context_settings=CONTEXT,
+    help="""Synchronize offline W&B run data to the cloud.
+
+    By default, scan the current directory for a `wandb`
```

Comment:
It actually prefers to check a `.wandb/` directory if that exists, and otherwise falls back to `wandb/`.